### ISSUE TITLE
Does not install from current master

### ISSUE THREAD

**Issue by Kdreval** [morinlab/GAMBLR.utils#11]
There is probably some missing comma or a bracket somewhere. This is what I get when installing from the current master:
```
✔  checking DESCRIPTION meta-information ...
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:28: unknown macro '\item'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:30: unknown macro '\item'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:32: unexpected section header '\value'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:35: unexpected section header '\description'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:38: unexpected section header '\details'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:44: unexpected section header '\examples'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/adjust_ploidy.Rd:55: unexpected END_OF_INPUT '
   '
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:45: unknown macro '\item'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:47: unknown macro '\item'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:49: unknown macro '\item'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:51: unknown macro '\item'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:53: unexpected section header '\value'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:56: unexpected section header '\description'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:59: unexpected section header '\details'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:68: unexpected section header '\examples'
   Warning: /tmp/Rtmpnv2QOI/Rbuild18ee466cb886f/GAMBLR.utils/man/cnvKompare.Rd:79: unexpected END_OF_INPUT '
   '
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
   Omitted ‘LazyData’ from DESCRIPTION
─  building ‘GAMBLR.utils_0.1.0.tar.gz’
   
Running /gsc/software/linux-x86_64-centos7/R-4.1.3/lib64/R/bin/R CMD INSTALL \
  /tmp/RtmpP2SDPz/GAMBLR.utils_0.1.0.tar.gz --install-tests 
* installing to library ‘/home/kdreval/R/x86_64-pc-linux-gnu-library/4.1’
* installing *source* package ‘GAMBLR.utils’ ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
** help
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:28: unknown macro '\item'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:30: unknown macro '\item'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:32: unexpected section header '\value'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:35: unexpected section header '\description'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:38: unexpected section header '\details'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:44: unexpected section header '\examples'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/adjust_ploidy.Rd:55: unexpected END_OF_INPUT '
'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:45: unknown macro '\item'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:47: unknown macro '\item'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:49: unknown macro '\item'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:51: unknown macro '\item'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:53: unexpected section header '\value'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:56: unexpected section header '\description'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:59: unexpected section header '\details'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:68: unexpected section header '\examples'
Warning: /tmp/RtmpSql4pn/R.INSTALL1904d4885504a/GAMBLR.utils/man/cnvKompare.Rd:79: unexpected END_OF_INPUT '
```

**Comment by Kdreval** [comment]
This also makes it non-installable through github
```
* building ‘GAMBLR.utils_0.1.0.tar.gz’

Loading custom .Rprofile for GAMBLR remote
ERROR: dependencies ‘ComplexHeatmap’, ‘GenomicRanges’, ‘IRanges’, ‘maftools’, ‘Rsamtools’, ‘SRAdb’ are not available for package ‘GAMBLR.utils’
* removing ‘/usr/share/miniconda/envs/dependencies/lib/R/library/GAMBLR.utils’
*
 ```

**Comment by Kdreval** [comment]
This is now solved in #12
